src/App.tsx

```diff
@@ -1,61 +1,120 @@
 import './App.scss';
+import { ChangeEvent, useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
+import { User } from './types/User';
+
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+export const todos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
 
 export const App = () => {
+  const [userName, setUserName] = useState(0);
```

Comment:
Why useName have a 0 initial value?

Reply:
Because 0 it's default value with string 'Choose a user'

Reply:
but it's not a name, it's `userId`